src/database/index.ts

```diff
@@ -0,0 +1 @@
+export { colors, fonts } from './database';
```

Comment:
did you forget to rename this folder? I guess database is not an appropriate to file name

Reply:
Yes updated to theme-data.ts now it makes more sense